yarn.lock

```diff
@@ -606,6 +606,7 @@
   version "7.8.3"
   resolved "https://registry.yarnpkg.com/@babel/plugin-syntax-optional-chaining/-/plugin-syntax-optional-chaining-7.8.3.tgz#4f69c2ab95167e0180cd5336613f8c5788f7d48a"
   integrity sha512-KoK9ErH1MBlCPxV0VANkXW2/dw4vlbGDrFgz8bmUsBGYkFRcbRwMh6cIJubdPrkxRwuGdtCk0v/wPTKbQgBjkg==
+<<<<<<< HEAD
```

Comment:
błąd przy mergowaniu, to trzeba usunąć. Pisałem na grupie jak radzić sobie z yarn.lock